puppetserver-standalone/Dockerfile

```diff
@@ -44,4 +44,14 @@ EXPOSE 8140
 ENTRYPOINT ["dumb-init", "/docker-entrypoint.sh"]
 CMD ["foreground" ]
 
+HEALTHCHECK --interval=10s --timeout=10s --retries=90 CMD \
+  curl --fail -H 'Accept: pson' \
+  --resolve 'puppet:8140:127.0.0.1' \
+  --cert   $(puppet config print hostcert) \
+  --key    $(puppet config print hostprivkey) \
+  --cacert $(puppet config print localcacert) \
+  https://puppet:8140/production/status/test \
```

Comment:
How about swapping the hard coded `production` for an ENV and setting that to default to production? This would be a sane default, but also allow users who don't have an environment named production to override the default at runtime without building a new image? Maybe setting `HEALTHCHECK_PUPPET_ENVIRONMENT` or similar?